package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.7.4",
+  "version": "6.8.4",
```

Comment:
The new version should be `7.0.0` since it's a breaking change and require a **major** version change.

https://semver.org/#summary

Reply:
Bumped to 7.0.0